libs/wallet-ui/src/components/code-window/code-window.tsx

```diff
@@ -8,16 +9,27 @@ export const CodeWindow = ({
   content: ReactNode
   text: string
 }) => {
+  const codeElRef = useRef<HTMLDivElement>(null)
+
+  // scroll to the dummy element when new logs arrive
```

Comment:
created an issue for this, so we have some context for this: https://github.com/vegaprotocol/vegawallet-ui/issues/179